jx_sqlite/expressions.py

```diff
@@ -144,7 +144,7 @@ def to_sql(self, schema, not_null=False, boolean=False):
                 if r.sql[t] == None:
                     acc.append("(" + l.sql[t] + ") IS NULL")
                 else:
-                    acc.append("((" + l.sql[t] + ") = (" + r.sql[t] + ") OR ((" + l.sql[t] + ") IS NULL AND (" + r.sql[t] + ") IS NULL))")
+                    acc.append("COALESCE((" + l.sql[t] + ") = (" + r.sql[t] + "), (" + l.sql[t] + ") IS NULL AND (" + r.sql[t] + ") IS NULL)")
```

Comment:
Since `a is null and b is null` appears to always return a not-null value *and* it has the correct value (false) when `a=b` returns null, `coalesce` seemed like a good choice.